bagging.md

```diff
@@ -1,22 +1,32 @@
 # Baggers
 
 - Getting set up as a Bagger
-  - Talk to your DataRescue guide to make sure you can become a Bagger.
+  - Apply to become a Bagger 
+    - By asking your DataRescue guide or by filling out [this form](XXX)
```

Comment:
I think we need to think about this exact application form workflow a bit more. If it is one per event, then certainly we don't want to list it right there. But if it is only one form for all events  then we can keep the link.